sql/src/SqlSelect.sk

```diff
@@ -303,7 +303,7 @@ class SelectEvaluator{
         RowValues::type,
         result.dirName,
       );
-      filtered = handle.filter(
+      _filtered = handle.filter(
```

Comment:
I don't understand this change; doesn't it result in the filter being ignored in this case?